pages/web/domains/api_domain_order/guide.it-it.md

```diff
@@ -668,8 +669,8 @@ This call also validates the configurations, like owner eligibility for a domain
 
 | Parameter                           | Required | Default | Description                                                                            |
 | ----------------------------------- | -------- | ------- | -------------------------------------------------------------------------------------- |
-| `autoPayWithPreferredPaymentMethod` | yes      | ""      | Used to automatically pay the purchase order with de default payment method of the nic |
-| `waiveRetractationPeriod`           | yes      | ""      | Required for a domain name, it is used to waive your right to retract                  |
+| `autoPayWithPreferredPaymentMethod` | no      | ""      | Used to automatically pay the purchase order with de default payment method of the nic |
+| `waiveRetractationPeriod`           | no      | ""      | Required for a domain name, it is used to waive your right to retract                  |
```

Comment:
description says it is required, are you sure about your change?

Reply:
Hi, 100% sure. Here the proof: the order 186056029 was created by API.